src/framework/audio/internal/worker/abstractaudiosource.h

```diff
@@ -32,11 +32,15 @@ class AbstractAudioSource : public IAudioSource
 
     virtual void setSampleRate(unsigned int sampleRate) override;
 
+    virtual bool isActive() const override;
+    virtual void setIsActive(bool arg) override;
```

Comment:
better without `virtual`

Reply:
yep, thanks

fixed in https://github.com/musescore/MuseScore/pull/8471/commits/4b672c8c28daa3a027ee891548c119f9bc981bf6